src/locales/sk.json

```diff
@@ -0,0 +1,174 @@
+            "hide_dock": "Skryť ikonu z Docku",
+            "fullscreen": "Spustiť režim celej obrazovky",
+            "tray": {
+                "show_tray": "Zobraziť ikonu v oznamovacej oblasti",
+                "show_window": "Zobraziť okno",
+                "toggle_minimize": "Prepnúť minimalizáciu do oznamovacej oblasti",
+                "start_minimize": "Spustiť minimalizované v oznamovacej oblasti",
+                "close": "Zavrieť do oznamovacej oblasti"
+            },
+            "escape": {
+                "title": "Správanie klávesy ESC",
+                "hide": "Skryje okno",
+                "clear": "Vyčistí vstup",
+                "default": "základné, keď sa nezobrazuje v oznamovacej oblasti"
+            },
+            "conversation": {
+                "title": "Zoznam konverzácií",
+                "timestamp": "Zobraziť čas konverzácie",
+                "last": "Zobraziť poslednú správu konverzácie",
+                "previous": "Predchádzajúca konverzácia",
+                "next": "Ďalšia konverzácia",
+                "select": "Vybrať konverzáciu",
+                "thumbnails": {
+                    "show": "Zobraziť náhľady",
+                    "only": "Zobraziť len náhľady",
+                    "animated": "Zobraziť animované náhľady"
+                }
+            },
+            "notification": {
+                "title": "Vyskakujúce upozornenia",
+                "show": "Zobraziť upozornenia",
+                "toggle": "Prepnúť upozornenia",
+                "message": "Zobraziť správu v upozorneniach",
+                "username": "Zobraziť užívateľské meno v upozorneniach",
+                "mute": "Vypnúť zvuk upozornení",
+                "avatar": "Zobraziť ikonu užívateľa v upozorneniach",
+                "icon": "Zobraziť YakYak ikonu v upozorneniach",
+                "custom_sound": "Použiť vlastný YakYak zvuk pre upozornenia"
+            },
+            "color_scheme": {
+                "title": "Farebná schéma",
+                "default": "Základná",
+                "blue": "Modrá",
+                "dark": "Tmavá",
+                "material": "Material"
+            },
+            "font": {
+                "title": "Veľkosť písma",
+                "extra_small": "Extra malé",
+                "small": "Malé",
+                "medium": "Stredné",
+                "large": "Veľké",
+                "extra_large": "Extra veľké"
+            },
+            "zoom": {
+                "in": "Priblížiť",
+                "out": "Oddialiť",
+                "reset": "Skutočná veľkosť"
+            }
+        },
+        "help": {
+            "title": "Pomocník",
+            "about": {
+                "title": "O aplikácii YakYak",
+                "newer": "Novšia verzia je k dispozícii. Je možné aktualizovat z %s na %s",
+                "license": "Licencia",
+                "authors": "Hlavní autori",
+                "contributors": "Prispievatelia"
```

Comment:
there's a string missing.

can you add a comma after `"contributors": "Prispievatelia"` and then translate "Open on Startup"

```
"contributors": "Prispievatelia",
"startup": "Open on Startup"
```